crates/biome_js_analyze/tests/specs/nursery/noStaticElementId/valid.jsx

```diff
@@ -0,0 +1,37 @@
+// should not generate diagnostics;
+function Foo() {
+	const id = useId();
+	return <div id={id}></div>;
+}
+
+function Foo() {
+	const id = useId();
+	return (
+		<div id={id}>
+			<div>bar</div>
+		</div>
+	);
+}
+
+function Foo() {
+	const id = useId();
+	return (
+		<div>
+			<div id={id}>bar</div>
+		</div>
+	);
+}
+
+function Foo() {
+	const id = crypto.randomUUID();
+	return <div id={id}></div>;
+}
+
+function Foo({ id }) {
+	return <div id={id}></div>;
+}
+
+function Foo() {
+	const id = useId();
+	return React.createElement("div", { id });
+}
```

Comment:
Can we add more test cases? For example, the following snippet shouldn't be triggered 

```js
createSomething("div", id)
```

This snippet should trigger the rule 
```js
import {createElement} from "react"
createElement("div", id)
```
Also, replacing `"react"` with something else shouldn't trigger the rule

Reply:
Thank you, I overlooked it. I added the test cases: https://github.com/biomejs/biome/pull/6082/commits/63f339149aaa630e7bc4512eb0d034069fcb81ce